src/CreatorToken.sol

```diff
@@ -28,6 +31,11 @@ contract CreatorToken is ERC721 {
     uint256 _adminFee
   );
 
+  modifier isNotAddressZero(address _address) {
```

Comment:
Since we're adding the address zero check (wasn't explicitly required, but I think it's fine to add) let's also add the check in the constructor (and associated tests).

Reply:
Added them here: https://github.com/showtime-xyz/creator-tokens/pull/9/commits/e2d09d3b8c2cf5ef735b091b4af37365a82000d1